.github/workflows/full-check.yml

```diff
@@ -4,10 +4,6 @@
 # build, though maven snapshots are not yet deployed.
 name: Full build
 on:
-  schedule:
```

Comment:
Note that with this removed, we will no longer publish to sonatype snapshots - the steps below use this check:
```
if: ${{ github.event_name == 'schedule' && github.repository_owner == 'gwtproject' && matrix.java-version == '21' }}
```
This check is far from perfect - but as `schedule` only runs on the default branch, we don't have to worry about pushes to `release/*` overwriting latest HEAD-SNAPSHOT. To finish this patch and let a `push` event replace the `schedule` event for publishing, we'll have to make sure the GWT_VERSION env var makes sense too.

Reply:
New version pushed that should run this only when pushed to `main`. As I understand there are currently no Sonatype snapshots for `release/*` branches so maybe we can keep it like that for now?